src/app/Main/Body/QueryPage/QueryPage-Layout.component.tsx

```diff
@@ -17,10 +17,10 @@ const subHeaderStyle = css`
     background-color: ${bfrPrimaryPalette[300]};
     box-sizing: border-box;
     box-shadow: 0 8px 6px -6px grey;
-    z-index: 10;
+    z-index: 1;
```

Comment:
Nope
This component should go like:
mainStyle is the parent and should establish a new stacking context. As documented use:
mainStyle: position: relative (also the default but it must be set to establish the stacking), z-index: 0
subHeaderStyle and queryPageStyle are your components you want to arrange in the stacking-context, so:
subHeaderStyle: z-index: 1
queryPageStyle: z-index: 0